apps/storybook-react/.storybook/preview.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import '@metamask/design-tokens/src/css/index.css';
+import '../../../packages/design-tokens/dist/styles.css';
```

Comment:
In Storybook's Vite setup, the relative path works because:
- Vite can resolve relative paths to files in the monorepo workspace
- The direct file access bypasses any package resolution issues
- The built CSS file contains all the processed styles ready for use

<img width="401" height="161" alt="Screenshot 2025-07-16 at 9 43 19 AM" src="https://github.com/user-attachments/assets/0fdf4c84-3251-4ec5-914e-1a417c3a66bb" />